ui/src/bootstrap.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
Is it necessarily to rename `index.js`?

Reply:
Yes, this is required when Module Federation is being used, so `bootstrap.js` is loaded asynchronously which allows the host app to wait for the remote components.